hconfig_utils/hconfig_get_private.F90

```diff
@@ -146,4 +164,104 @@ function make_fmt(descriptor) result(fmt)
 
    end function make_fmt
 
+#include "mapl_hconfig_macros.h"
+
+#define _SUB_ mapl_get_value_I4
+#define _FTYPE_ integer(kind=ESMF_KIND_I4)
+#define _ESMF_FUNC_ ESMF_HConfigAsI4
+#define _TYPESTRING_ "I4"
+#define _EDIT_DESCRIPTOR_ "G0"
```

Comment:
Since these are not user-level, we may want to future proof this layer by using tokens that are not "reserved'' by the language.  Fortran will soon reserve almost everything that starts with a leading underscore. (unfortunately)

No action now, but think about it.

Reply:
Makes sense. I want to use tokens that are not valid Fortran identifiers for macros, so that compile will fail if the tokens are not replaced. I guess I could use a valid identifier. However, if I use `FOO_` and there is a variable `foo_`, it might not fail even though I don't want it to use the variable `foo_`.

What would be a non-reserved, non-valid token pattern?